webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
- [ ] It will be nice to use ES6 `export` over `module.exports`

Reply:
Hi @wandji20,
Please, if we use `export` over `module.exports`, it will show an error when we run this line of code in the terminal `npx webpack --config webpack.config.js`
Thanks for the feedback
![Screenshot_2021-10-15_14_29_29](https://user-images.githubusercontent.com/71507900/137488975-dae686c3-3698-4d7b-a3b3-0b93898be0f5.png)